components/Forms/CurrencyField.tsx

```diff
@@ -36,22 +36,23 @@ export const CurrencyField: React.VFC<CurrencyFieldProps> = observer(
 
     return (
       <div>
-        <label
-          htmlFor={name}
-          aria-label={name}
-          data-testid="currency-input-label"
-          className="text-content font-bold inline-block mb-2.5"
-        >
-          {required && <span className="text-danger">*</span>} {label}
-          <span>
+        <div className="mb-2.5">
+          <label
+            htmlFor={name}
+            aria-label={name}
+            data-testid="currency-input-label"
+            className="text-content font-bold inline"
+          >
+            {required && <span className="text-danger">*</span>} {label}
+          </label>
+          <div>
             {required && (
               <span className="text-danger ml-1">({requiredText})</span>
             )}
             <Tooltip field={name} />
-          </span>
-        </label>
+          </div>
+        </div>
         {error && <ErrorLabel errorMessage={error} />}
```

Comment:
This fix pulls the tooltip out of the `<label>` so when a user clicks the tooltip the focus will correctly change!